### ISSUE TITLE
Rails new command broken

### ISSUE THREAD

**Issue by cdurante** [refinery/refinerycms#3081]
Just tried pulling this down, getting errors with *speakurl-rails* it appears. 

I've tried reinstalling ruby, using 2.1.0, 2.2.0, and 2.2.3 in conjunction with Rails 4.2.5, 4.2, and 5(not that I expected that to work). 

Any subsequent commands are blocked by the NoMethodError (i.e. rake db:... rails server, etc.)

Been banging my head against the terminal for hours... Is there an obvious mistake that I am making or is there a dependency issue within refineryCMS? 

**Below is the output generated with this combination of Ruby/Rails:**
```console
$ ruby -v 
ruby 2.0.0p648 (2015-12-16 revision 53162) [x86_64-darwin15.2.0]
$ rails -v
Rails 4.2.5
```

**Rails new command:**
```console
$ rails new cfinery -m http://refinerycms.com/t/edge
      create  
      create  README.rdoc
      create  Rakefile
      create  config.ru
      create  .gitignore
      create  Gemfile
      create  app
      create  app/assets/javascripts/application.js
      create  app/assets/stylesheets/application.css
      create  app/controllers/application_controller.rb
      create  app/helpers/application_helper.rb
      create  app/views/layouts/application.html.erb
      create  app/assets/images/.keep
      create  app/mailers/.keep
      create  app/models/.keep
      create  app/controllers/concerns/.keep
      create  app/models/concerns/.keep
      create  bin
      create  bin/bundle
      create  bin/rails
      create  bin/rake
      create  bin/setup
      create  config
      create  config/routes.rb
      create  config/application.rb
      create  config/environment.rb
      create  config/secrets.yml
      create  config/environments
      create  config/environments/development.rb
      create  config/environments/production.rb
      create  config/environments/test.rb
      create  config/initializers
      create  config/initializers/assets.rb
      create  config/initializers/backtrace_silencers.rb
      create  config/initializers/cookies_serializer.rb
      create  config/initializers/filter_parameter_logging.rb
      create  config/initializers/inflections.rb
      create  config/initializers/mime_types.rb
      create  config/initializers/session_store.rb
      create  config/initializers/wrap_parameters.rb
      create  config/locales
      create  config/locales/en.yml
      create  config/boot.rb
      create  config/database.yml
      create  db
      create  db/seeds.rb
      create  lib
      create  lib/tasks
      create  lib/tasks/.keep
      create  lib/assets
      create  lib/assets/.keep
      create  log
      create  log/.keep
      create  public
      create  public/404.html
      create  public/422.html
      create  public/500.html
      create  public/favicon.ico
      create  public/robots.txt
      create  test/fixtures
      create  test/fixtures/.keep
      create  test/controllers
      create  test/controllers/.keep
      create  test/mailers
      create  test/mailers/.keep
      create  test/models
      create  test/models/.keep
      create  test/helpers
      create  test/helpers/.keep
      create  test/integration
      create  test/integration/.keep
      create  test/test_helper.rb
      create  tmp/cache
      create  tmp/cache/assets
      create  vendor/assets/javascripts
      create  vendor/assets/javascripts/.keep
      create  vendor/assets/stylesheets
      create  vendor/assets/stylesheets/.keep
       apply  http://refinerycms.com/t/edge
      append    Gemfile
      append    Gemfile
         run    bundle install from "."
Fetching https://github.com/refinery/refinerycms
Fetching gem metadata from https://rubygems.org/...........
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies......
Using rake 10.4.2
Using i18n 0.7.0
Using json 1.8.3
Using minitest 5.8.3
Using thread_safe 0.3.5
Using builder 3.2.2
Using erubis 2.7.0
Using mini_portile2 2.0.0
Using rack 1.6.4
Using mime-types 2.99
Using arel 6.0.3
Using acts_as_indexed 0.8.3
Using addressable 2.4.0
Using babosa 1.0.2
Using bcrypt 3.1.10
Using debug_inspector 0.0.2
Using bundler 1.11.2
Using byebug 8.2.1
Using coffee-script-source 1.10.0
Using execjs 2.6.0
Using thor 0.19.1
Using concurrent-ruby 1.0.0
Using orm_adapter 0.5.0
Using multi_json 1.11.2
Using sass 3.4.20
Using tilt 2.0.1
Using truncate_html 0.9.3
Using will_paginate 3.0.7
Using zilch-authorisation 0.0.1
Using spring 1.6.0
Using sqlite3 1.3.11
Using rdoc 4.2.0
Using tzinfo 1.2.2
Using nokogiri 1.6.7.1
Using rack-test 0.6.3
Using warden 1.2.4
Using mail 2.6.3
Using binding_of_caller 0.7.2
Using coffee-script 2.4.1
Using uglifier 2.7.2
Using sprockets 3.5.2
Using dragonfly 1.0.12
Using sdoc 0.4.1
Using activesupport 4.2.5
Using loofah 2.0.3
Using rails-deprecated_sanitizer 1.0.3
Using globalid 0.3.6
Using activemodel 4.2.5
Using jbuilder 2.3.2
Using rails-html-sanitizer 1.0.2
Using rails-dom-testing 1.0.7
Using activejob 4.2.5
Using activerecord 4.2.5
Using actionview 4.2.5
Using awesome_nested_set 3.0.2
Using friendly_id 5.1.0
Using globalize 5.0.1
Using actionpack 4.2.5
Using actionmailer 4.2.5
Using railties 4.2.5
Using sprockets-rails 3.0.0
Using routing-filter 0.5.1
Using coffee-rails 4.1.1
Using decorators 2.0.1
Using responders 2.1.1
Using jquery-rails 4.0.5
Using jquery-ui-rails 5.0.5
Using quiet_assets 1.1.0
Using rails-i18n 4.0.7
Using seo_meta 2.0.0.rc.1
Using speakingurl-rails 1.1.5
Using rails 4.2.5
Using sass-rails 5.0.4
Using web-console 2.2.1
Using turbolinks 2.5.3
Using devise 3.5.3
Using refinerycms-i18n 3.0.1
Using refinerycms-core 3.0.0 from https://github.com/refinery/refinerycms (at master@028b3b7)
Using refinerycms-images 3.0.0 from https://github.com/refinery/refinerycms (at master@028b3b7)
Using refinerycms-pages 3.0.0 from https://github.com/refinery/refinerycms (at master@028b3b7)
Using refinerycms-resources 3.0.0 from https://github.com/refinery/refinerycms (at master@028b3b7)
Using refinerycms-acts-as-indexed 2.0.1
Using refinerycms-authentication-devise 1.0.4
Using refinerycms-wymeditor 1.0.6
Using refinerycms 3.0.0 from https://github.com/refinery/refinerycms (at master@028b3b7)
Bundle complete! 17 Gemfile dependencies, 85 gems now installed.`
Use `bundle show [gemname]` to see where a bundled gem is installed.
        rake    db:create
    generate    refinery:cms --fresh-installation new cfinery -m http://refinerycms.com/t/edge
/Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/speakingurl-rails-1.1.5/lib/speakingurl-rails.rb:8:in `block in <class:Railtie>': undefined method `prepend_path' for nil:NilClass (NoMethodError)
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/initializable.rb:30:in `instance_exec'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/initializable.rb:30:in `run'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/initializable.rb:55:in `block in run_initializers'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/2.0.0/tsort.rb:150:in `block in tsort_each'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/2.0.0/tsort.rb:183:in `block (2 levels) in each_strongly_connected_component'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/2.0.0/tsort.rb:219:in `each_strongly_connected_component_from'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/2.0.0/tsort.rb:182:in `block in each_strongly_connected_component'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/2.0.0/tsort.rb:180:in `each'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/2.0.0/tsort.rb:180:in `each_strongly_connected_component'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/2.0.0/tsort.rb:148:in `tsort_each'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/initializable.rb:54:in `run_initializers'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/application.rb:352:in `initialize!'
	from /Users/Caleb/cfinery/config/environment.rb:5:in `<top (required)>'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `require'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `block in require'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `require'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/application.rb:328:in `require_environment!'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:142:in `require_application_and_environment!'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:128:in `generate_or_destroy'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:50:in `generate'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /Users/Caleb/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/railties-4.2.5/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'
    ============================================================================
    Your new Refinery CMS application is now running on edge and mounted to /.
  ============================================================================
         run  bundle install
Using rake 10.4.2
Using i18n 0.7.0
Using json 1.8.3
Using minitest 5.8.3
Using thread_safe 0.3.5
Using builder 3.2.2
Using erubis 2.7.0
Using mini_portile2 2.0.0
Using rack 1.6.4
Using mime-types 2.99
Using arel 6.0.3
Using acts_as_indexed 0.8.3
Using addressable 2.4.0
Using babosa 1.0.2
Using bcrypt 3.1.10
Using debug_inspector 0.0.2
Using byebug 8.2.1
Using coffee-script-source 1.10.0
Using execjs 2.6.0
Using thor 0.19.1
Using concurrent-ruby 1.0.0
Using orm_adapter 0.5.0
Using multi_json 1.11.2
Using bundler 1.11.2
Using sass 3.4.20
Using tilt 2.0.1
Using truncate_html 0.9.3
Using will_paginate 3.0.7
Using zilch-authorisation 0.0.1
Using spring 1.6.0
Using sqlite3 1.3.11
Using rdoc 4.2.0
Using tzinfo 1.2.2
Using nokogiri 1.6.7.1
Using rack-test 0.6.3
Using warden 1.2.4
Using mail 2.6.3
Using binding_of_caller 0.7.2
Using coffee-script 2.4.1
Using uglifier 2.7.2
Using sprockets 3.5.2
Using dragonfly 1.0.12
Using sdoc 0.4.1
Using activesupport 4.2.5
Using loofah 2.0.3
Using rails-deprecated_sanitizer 1.0.3
Using globalid 0.3.6
Using activemodel 4.2.5
Using jbuilder 2.3.2
Using rails-html-sanitizer 1.0.2
Using rails-dom-testing 1.0.7
Using activejob 4.2.5
Using activerecord 4.2.5
Using actionview 4.2.5
Using awesome_nested_set 3.0.2
Using friendly_id 5.1.0
Using globalize 5.0.1
Using actionpack 4.2.5
Using actionmailer 4.2.5
Using railties 4.2.5
Using sprockets-rails 3.0.0
Using routing-filter 0.5.1
Using coffee-rails 4.1.1
Using decorators 2.0.1
Using responders 2.1.1
Using jquery-rails 4.0.5
Using jquery-ui-rails 5.0.5
Using quiet_assets 1.1.0
Using rails-i18n 4.0.7
Using seo_meta 2.0.0.rc.1
Using speakingurl-rails 1.1.5
Using rails 4.2.5
Using sass-rails 5.0.4
Using web-console 2.2.1
Using turbolinks 2.5.3
Using devise 3.5.3
Using refinerycms-i18n 3.0.1
Using refinerycms-core 3.0.0 from https://github.com/refinery/refinerycms (at master@028b3b7)
Using refinerycms-images 3.0.0 from https://github.com/refinery/refinerycms (at master@028b3b7)
Using refinerycms-pages 3.0.0 from https://github.com/refinery/refinerycms (at master@028b3b7)
Using refinerycms-resources 3.0.0 from https://github.com/refinery/refinerycms (at master@028b3b7)
Using refinerycms-acts-as-indexed 2.0.1
Using refinerycms-authentication-devise 1.0.4
Using refinerycms-wymeditor 1.0.6
Using refinerycms 3.0.0 from https://github.com/refinery/refinerycms (at master@028b3b7)
Bundle complete! 17 Gemfile dependencies, 85 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.
         run  bundle exec spring binstub --all
* bin/rake: spring inserted
* bin/rails: spring inserted
```

**Gemfile**

```ruby
source 'https://rubygems.org'


# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '4.2.5'
# Use sqlite3 as the database for Active Record
gem 'sqlite3'
# Use SCSS for stylesheets
gem 'sass-rails', '~> 5.0'
# Use Uglifier as compressor for JavaScript assets
gem 'uglifier', '>= 1.3.0'
# Use CoffeeScript for .coffee assets and views
gem 'coffee-rails', '~> 4.1.0'
# See https://github.com/rails/execjs#readme for more supported runtimes
# gem 'therubyracer', platforms: :ruby

# Use jquery as the JavaScript library
gem 'jquery-rails'
# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
gem 'turbolinks'
# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
gem 'jbuilder', '~> 2.0'
# bundle exec rake doc:rails generates the API under doc/api.
gem 'sdoc', '~> 0.4.0', group: :doc

# Use ActiveModel has_secure_password
# gem 'bcrypt', '~> 3.1.7'

# Use Unicorn as the app server
# gem 'unicorn'

# Use Capistrano for deployment
# gem 'capistrano-rails', group: :development

group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug'
end

group :development do
  # Access an IRB console on exception pages or by using <%= console %> in views
  gem 'web-console', '~> 2.0'

  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
end


gem 'refinerycms', git: 'https://github.com/refinery/refinerycms', branch: 'master'

gem 'quiet_assets', group: :development

# Add support for searching inside Refinery's admin interface.
gem 'refinerycms-acts-as-indexed', ['~> 2.0', '>= 2.0.0']

# Add support for Refinery's custom fork of the visual editor WYMeditor.
gem 'refinerycms-wymeditor', ['~> 1.0', '>= 1.0.6']

# The default authentication adapter
gem 'refinerycms-authentication-devise', '~> 1.0'
```

**Gemfile.lock**

```ruby
GIT
  remote: https://github.com/refinery/refinerycms
  revision: 028b3b70f14773ad0a1c7e113f418e218e5f417e
  branch: master
  specs:
    refinerycms (3.0.0)
      refinerycms-core (= 3.0.0)
      refinerycms-images (= 3.0.0)
      refinerycms-pages (= 3.0.0)
      refinerycms-resources (= 3.0.0)
    refinerycms-core (3.0.0)
      actionpack (>= 4.2.3, < 5.0)
      activerecord (>= 4.2.3, < 5.0)
      awesome_nested_set (~> 3.0.0)
      coffee-rails (~> 4.0, >= 4.0.0)
      decorators (~> 2.0.0)
      jquery-rails (>= 2.3.0)
      jquery-ui-rails (~> 5.0.0)
      railties (>= 4.2.3, < 5.0)
      refinerycms-i18n (~> 3.0.0)
      sass-rails (>= 4.0, < 5.1)
      truncate_html (~> 0.9)
      will_paginate (~> 3.0.2)
      zilch-authorisation
    refinerycms-images (3.0.0)
      dragonfly (~> 1.0.0)
      globalize (>= 4.0.0, < 5.2)
      refinerycms-core (= 3.0.0)
    refinerycms-pages (3.0.0)
      awesome_nested_set (~> 3.0.0)
      babosa (!= 0.3.6)
      friendly_id (~> 5.1.0)
      globalize (>= 4.0.0, < 5.2)
      refinerycms-core (= 3.0.0)
      seo_meta (~> 2.0.0.rc.1)
      speakingurl-rails (~> 1.1.2)
    refinerycms-resources (3.0.0)
      acts_as_indexed (~> 0.8.0)
      dragonfly (~> 1.0.0)
      globalize (>= 4.0.0, < 5.2)
      refinerycms-core (= 3.0.0)

GEM
  remote: https://rubygems.org/
  specs:
    actionmailer (4.2.5)
      actionpack (= 4.2.5)
      actionview (= 4.2.5)
      activejob (= 4.2.5)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 1.0, >= 1.0.5)
    actionpack (4.2.5)
      actionview (= 4.2.5)
      activesupport (= 4.2.5)
      rack (~> 1.6)
      rack-test (~> 0.6.2)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (4.2.5)
      activesupport (= 4.2.5)
      builder (~> 3.1)
      erubis (~> 2.7.0)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    activejob (4.2.5)
      activesupport (= 4.2.5)
      globalid (>= 0.3.0)
    activemodel (4.2.5)
      activesupport (= 4.2.5)
      builder (~> 3.1)
    activerecord (4.2.5)
      activemodel (= 4.2.5)
      activesupport (= 4.2.5)
      arel (~> 6.0)
    activesupport (4.2.5)
      i18n (~> 0.7)
      json (~> 1.7, >= 1.7.7)
      minitest (~> 5.1)
      thread_safe (~> 0.3, >= 0.3.4)
      tzinfo (~> 1.1)
    acts_as_indexed (0.8.3)
    addressable (2.4.0)
    arel (6.0.3)
    awesome_nested_set (3.0.2)
      activerecord (>= 4.0.0, < 5)
    babosa (1.0.2)
    bcrypt (3.1.10)
    binding_of_caller (0.7.2)
      debug_inspector (>= 0.0.1)
    builder (3.2.2)
    byebug (8.2.1)
    coffee-rails (4.1.1)
      coffee-script (>= 2.2.0)
      railties (>= 4.0.0, < 5.1.x)
    coffee-script (2.4.1)
      coffee-script-source
      execjs
    coffee-script-source (1.10.0)
    concurrent-ruby (1.0.0)
    debug_inspector (0.0.2)
    decorators (2.0.1)
      railties (>= 4.0.0, < 5.0.0)
    devise (3.5.3)
      bcrypt (~> 3.0)
      orm_adapter (~> 0.1)
      railties (>= 3.2.6, < 5)
      responders
      thread_safe (~> 0.1)
      warden (~> 1.2.3)
    dragonfly (1.0.12)
      addressable (~> 2.3)
      multi_json (~> 1.0)
      rack (>= 1.3.0)
    erubis (2.7.0)
    execjs (2.6.0)
    friendly_id (5.1.0)
      activerecord (>= 4.0.0)
    globalid (0.3.6)
      activesupport (>= 4.1.0)
    globalize (5.0.1)
      activemodel (>= 4.2.0, < 4.3)
      activerecord (>= 4.2.0, < 4.3)
    i18n (0.7.0)
    jbuilder (2.3.2)
      activesupport (>= 3.0.0, < 5)
      multi_json (~> 1.2)
    jquery-rails (4.0.5)
      rails-dom-testing (~> 1.0)
      railties (>= 4.2.0)
      thor (>= 0.14, < 2.0)
    jquery-ui-rails (5.0.5)
      railties (>= 3.2.16)
    json (1.8.3)
    loofah (2.0.3)
      nokogiri (>= 1.5.9)
    mail (2.6.3)
      mime-types (>= 1.16, < 3)
    mime-types (2.99)
    mini_portile2 (2.0.0)
    minitest (5.8.3)
    multi_json (1.11.2)
    nokogiri (1.6.7.1)
      mini_portile2 (~> 2.0.0.rc2)
    orm_adapter (0.5.0)
    quiet_assets (1.1.0)
      railties (>= 3.1, < 5.0)
    rack (1.6.4)
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (4.2.5)
      actionmailer (= 4.2.5)
      actionpack (= 4.2.5)
      actionview (= 4.2.5)
      activejob (= 4.2.5)
      activemodel (= 4.2.5)
      activerecord (= 4.2.5)
      activesupport (= 4.2.5)
      bundler (>= 1.3.0, < 2.0)
      railties (= 4.2.5)
      sprockets-rails
    rails-deprecated_sanitizer (1.0.3)
      activesupport (>= 4.2.0.alpha)
    rails-dom-testing (1.0.7)
      activesupport (>= 4.2.0.beta, < 5.0)
      nokogiri (~> 1.6.0)
      rails-deprecated_sanitizer (>= 1.0.1)
    rails-html-sanitizer (1.0.2)
      loofah (~> 2.0)
    rails-i18n (4.0.7)
      i18n (~> 0.7)
      railties (~> 4.0)
    railties (4.2.5)
      actionpack (= 4.2.5)
      activesupport (= 4.2.5)
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rake (10.4.2)
    rdoc (4.2.0)
    refinerycms-acts-as-indexed (2.0.1)
      acts_as_indexed (~> 0.8.0)
      decorators (~> 2.0.0)
      refinerycms-core (~> 3.0.0)
    refinerycms-authentication-devise (1.0.4)
      actionmailer (~> 4.2, >= 4.2.0)
      devise (~> 3.0, >= 3.2.4)
      friendly_id (~> 5.1.0)
      refinerycms-core (~> 3.0, >= 3.0.0)
    refinerycms-i18n (3.0.1)
      rails-i18n (~> 4.0.1)
      routing-filter (>= 0.4.0)
    refinerycms-wymeditor (1.0.6)
      refinerycms-core (~> 3.0, >= 3.0.0)
    responders (2.1.1)
      railties (>= 4.2.0, < 5.1)
    routing-filter (0.5.1)
      actionpack (~> 4.2)
      activesupport (~> 4.2)
    sass (3.4.20)
    sass-rails (5.0.4)
      railties (>= 4.0.0, < 5.0)
      sass (~> 3.1)
      sprockets (>= 2.8, < 4.0)
      sprockets-rails (>= 2.0, < 4.0)
      tilt (>= 1.1, < 3)
    sdoc (0.4.1)
      json (~> 1.7, >= 1.7.7)
      rdoc (~> 4.0)
    seo_meta (2.0.0.rc.1)
      railties (>= 3.0.0)
    speakingurl-rails (1.1.5)
      railties (>= 3.1)
    spring (1.6.0)
    sprockets (3.5.2)
      concurrent-ruby (~> 1.0)
      rack (> 1, < 3)
    sprockets-rails (3.0.0)
      actionpack (>= 4.0)
      activesupport (>= 4.0)
      sprockets (>= 3.0.0)
    sqlite3 (1.3.11)
    thor (0.19.1)
    thread_safe (0.3.5)
    tilt (2.0.1)
    truncate_html (0.9.3)
    turbolinks (2.5.3)
      coffee-rails
    tzinfo (1.2.2)
      thread_safe (~> 0.1)
    uglifier (2.7.2)
      execjs (>= 0.3.0)
      json (>= 1.8.0)
    warden (1.2.4)
      rack (>= 1.0)
    web-console (2.2.1)
      activemodel (>= 4.0)
      binding_of_caller (>= 0.7.2)
      railties (>= 4.0)
      sprockets-rails (>= 2.0, < 4.0)
    will_paginate (3.0.7)
    zilch-authorisation (0.0.1)

PLATFORMS
  ruby

DEPENDENCIES
  byebug
  coffee-rails (~> 4.1.0)
  jbuilder (~> 2.0)
  jquery-rails
  quiet_assets
  rails (= 4.2.5)
  refinerycms!
  refinerycms-acts-as-indexed (~> 2.0, >= 2.0.0)
  refinerycms-authentication-devise (~> 1.0)
  refinerycms-wymeditor (~> 1.0, >= 1.0.6)
  sass-rails (~> 5.0)
  sdoc (~> 0.4.0)
  spring
  sqlite3
  turbolinks
  uglifier (>= 1.3.0)
  web-console (~> 2.0)

BUNDLED WITH
   1.11.2
```

**Comment by cdurante** [comment]
overriding the sprockets-rails version seems to work:

```ruby
gem 'sprockets-rails', '2.3.3'
```

Compatibility with 3.0.0 in the pipeline?

**Comment by mwenger1** [comment]
:+1:

**Comment by bricesanchez** [comment]
I've created a pull request to fix this issue on speakingurl repo :
https://github.com/pid/speakingurl/pull/70

And i've proposed a temp fix on Refinery CMS : 
https://github.com/refinery/refinerycms/pull/3080

Refinery CMS seems to be compatible with sprockets-rails 3.0 when speakingurl will be fixed.

**Comment by ahmgeek** [comment]
+1

**Comment by pid** [comment]
I released speakingurl v8.0.0  ;-)
sorry for the delay, hope it works for you now

**Comment by parndt** [comment]
Thanks @pid

#3084
